cypress/e2e/inventory/search/browse-contributors-linked.cy.js

```diff
@@ -0,0 +1,130 @@
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcAuthC359596_1.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      naturalId: 'n831692600011181',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcAuthC359596_2.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      naturalId: 'n831692600011182',
+      numOfRecords: 1,
+    },
+  ];
+
+  const tagInfo = [
+    {
+      rowIndex: 12,
+      authNaturalId: 'n831692600011181',
+    },
+    {
+      rowIndex: 13,
+      authNaturalId: 'n831692600011182',
+    }
+  ];
+
+  const createdRecordIDs = [];
+
+  before('Creating data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+    ]).then(createdUserProperties => {
+      testData.userProperties = createdUserProperties;
+
+      marcFiles.forEach(marcFile => {
+        cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+          DataImport.uploadFile(marcFile.marc, marcFile.fileName);
+          JobProfiles.waitLoadingList();
+          JobProfiles.searchJobProfileForImport(marcFile.jobProfileToRun);
+          JobProfiles.runImportFile();
+          JobProfiles.waitFileIsImported(marcFile.fileName);
+          Logs.checkStatusOfJobProfile('Completed');
+          Logs.openFileDetails(marcFile.fileName);
+          for (let i = 0; i < marcFile.numOfRecords; i++) {
+            Logs.getCreatedItemsID(i).then(link => {
+              createdRecordIDs.push(link.split('/')[5]);
+            });
+          }
+        });
+      });
+
+      cy.visit(TopMenu.inventoryPath).then(() => {
+        InventoryInstances.waitContentLoading();
+        InventoryInstance.searchByTitle(createdRecordIDs[0]);
+        InventoryInstances.selectInstance();
+        InventoryInstance.editMarcBibliographicRecord();
+        tagInfo.forEach(tag => {
+          QuickMarcEditor.clickLinkIconInTagField(tag.rowIndex);
+          MarcAuthorities.switchToSearch();
+          InventoryInstance.verifySelectMarcAuthorityModal();
+          InventoryInstance.verifySearchOptions();
+          InventoryInstance.searchResultsWithOption('Identifier (all)', tag.authNaturalId);
+          InventoryInstance.clickLinkButton();
+          InventoryInstance.closeDetailsView();
+          InventoryInstance.closeFindAuthorityModal();
+          // wait for a modal to fully close
+          cy.wait(1500);
+        });
+        QuickMarcEditor.pressSaveAndClose();
+        InventoryInstance.waitLoading();
+      });
+    });
+  });
+
+  beforeEach('Login to the application', () => {
+    cy.login(testData.userProperties.username, testData.userProperties.password, { path: TopMenu.inventoryPath, waiter: InventoryInstances.waitContentLoading });
```

Comment:
If we have just one test in the file we can move cy.login() to "before" hook.

Reply:
Moved as proposed